tiledb/dataframe_.py

```diff
@@ -363,14 +377,15 @@ def create_dims(
 
         # get the FilterList, if any
         if isinstance(filters, dict):
-            if name in filters:
-                dim_filters = filters[name]
-            else:
-                dim_filters = None
-        elif filters is not None:
+            dim_filters = filters.get(name, None)
+        elif isinstance(filters, list):
+            dim_filters = tiledb.FilterList(filters)
+        elif isinstance(filters, tiledb.FilterList):
             dim_filters = filters
-        else:
+        elif filters or filters is None:
```

Comment:
Shouldn't this condition be just `elif filters is None`?

Reply:
#568